internal/service/service.go

```diff
@@ -62,6 +68,12 @@ func (s *Service) Routes() *mux.Router {
 	r := mux.NewRouter()
 	r.HandleFunc("/", s.logHandler(s.rootRequest))
 	r.HandleFunc("/query", s.logHandler(s.queryRequest))
+	if s.EnableDebug {
+		r.HandleFunc("/debug/pprof/", pprof.Index)
```

Comment:
`pprof.Index` should handle subpaths of "/debug/pprof/", so I'd expect these 3 lines can be condensed down to one.

Reply:
That was my expectation too, I commented out those 2 lines after initial test, it wasn't working as expected, so I put thos e lines back.